assignments/objects.js

```diff
@@ -19,27 +19,85 @@ const example = {
 
 // Write your intern objects here:
 
+const intern = [ 
+  {
+    "id": 1,
+    "email": "mmelloy0@psu.edu",
+    "name": "Mitzi",
+    "gender": "F"
+  },
+  {
+    "id": 2,
+    "email": "kdiben1@tinypic.com",
+    "name": "Kennan",
+    "gender": "M"
+  },
+  {
+    "id": 3,
+    "email": "gmartinson3@illinois.edu",
+    "name": "Keven",
+    "gender": "M"
+  },
+  {
+    "id": 4,
+    "email": "gmartinson3@illinois.edu",
+    "name": "Gannie",
+    "gender": "M"
+  },
+  {
+    "id": 5,
+    "email": "adaine5@samsung.com",
+    "name": "Antonietta",
+    "gender": "F"
+  }
+  ];
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(intern[0].name);
+
 // Kennan's ID
 
+console.log(intern[1].id);
+
 // Keven's email
 
+console.log(intern[2].email);
+
 // Gannie's name
 
+console.log(intern[3].name);
+
 // Antonietta's Gender
 
+console.log(intern[4].gender);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+let Kennan = {
+  speak: function() {
+    console.log(`Hello, my name is ${intern[1].name} `);
+  }
+}  
+ console.log(Kennan.speak());
+
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
 
+let antonietta = {
+  multiplyNums: function(a,b) {
+  return 3 * 4;
+}
+}  
+
+console.log(antonietta.multiplyNums(3,4));
+
 // === Great work! === Head over to the the arrays.js file or take a look at the stretch challenge
```

Comment:
I like what you did here! You put each object inside an array rather than have separate objects. It all works out either way though. Great job!

Reply:
Not sure if I did my review right... This project is good! No changes necessary :)